ibc-clients/ics08-wasm/types/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "ibc-client-wasm-types"
+version 	 = { workspace = true }
+authors 	 = { workspace = true }
+license 	 = { workspace = true }
+rust-version = { workspace = true }
+edition 	 = { workspace = true }
+
+[dependencies]
+# external dependencies
+base64          = { version = "0.21", default-features = false, features = ["alloc"] }
+bytes           = { version = "1.2.1", default-features = false }
+displaydoc      = { workspace = true }
+serde 			= { workspace = true , optional = true }
+cosmwasm-schema = { version = "1.4.1", default-features = false, optional = true }
+
+# ibc dependencies
+ibc-core-client     = { workspace = true, default-features = false }
+ibc-core-host-types = { workspace = true, default-features = false }
+ibc-primitives    	= { workspace = true, default-features = false }
+ibc-proto 		    = { workspace = true, default-features = false }
+
+[features]
+default = ["std"]
+std = [
+	"ibc-core-client/std",
+	"ibc-core-host-types/std",
+	"ibc-proto/std",
+	"base64/std",
+	"bytes/std",
+	"serde/std"
+]
+serde = [
+	"ibc-core-client/serde",
+	"ibc-core-host-types/serde",
+	"ibc-primitives/serde",
+	"ibc-proto/serde",
+	"dep:serde"
+]
+schema = [
+	"ibc-core-client/schema",
+	"ibc-core-host-types/schema",
+	"ibc-primitives/schema",
+	"ibc-proto/json-schema",
+	"serde",
+	"std"
+]
+cosmwasm = [
+	"cosmwasm-schema",
+	"schema"
+]
```

Comment:
This file has a mix of tabs and spaces. That's why the `=`s don't align.

Reply:
[952dc8e](https://github.com/cosmos/ibc-rs/pull/1031/commits/952dc8e71ab75743fc56ae54fde39abf01a78512)